lib/index.js

```diff
@@ -21,17 +21,14 @@ var SystemEOL = require('os').EOL;
  */
 
 var Exec = module.exports = exports = function (args, callback) {
-  if (Array.isArray(args)) {
-    args = args.join(' ');
-  }
+  var spawn = ChildProcess.spawn;
```

Comment:
hi @honzajavorek thanks for your reivew! I just took a look at cross-spawn, and the issues that it solves will not be an issue in our case:

Node has issues when using spawn on Windows:

- It ignores PATHEXT
- It does not support shebangs
- No options.shell support on node < v6
- It does not allow you to run del or dir